docs/release-notes/8437-auth-refactor.rst

```diff
@@ -0,0 +1,18 @@
+:orphan:
+
+**Removed Features**
+
+-  **Breaking Change** We removed the accidentally-exposed Session object from the
+   ``det.experimental.client`` namespace. It was never meant to be a public API and it was not
+   documented in :ref:`python-sdk`, but it was exposed in that namespace nonetheless. It was also
+   available as a deprecated legacy alias, ``det.experimental.Session``. It is expected that most
+   users use the Python SDK normally and are unaffected by this change, since the
+   ``det.experimental.client``'s ``login()`` and ``Determined()`` are unaffected.
+
+-  **Breaking Change** Add a new requirement for runtime configurations that there be a writable
```

Comment:
> limited support for such containers

limited support for containers without a writeable ``$HOME``

> this could affect a users

this could affect users

(suggestions)